Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -10911,7 +10915,14 @@ - (UITargetedPreview *)_createTargetedContextMenuHintPreviewIfPossible
     }
 
     if (!targetedPreview) {
-        auto boundsForFallbackPreview = self.positionInformationHasImageOverlayDataDetector ? _positionInformation.dataDetectorBounds : _positionInformation.bounds;
+        auto boundsForFallbackPreview = [&]() {
```

Comment:
I don't think the () is needed

Reply:
You likely need it for any non empty closure block... until 6/10!

Reply:
@rr-codes is correct!

Reply:
s/closure/capture